lib/absinthe/plug.ex

```diff
@@ -353,6 +353,36 @@ defmodule Absinthe.Plug do
     Plug.Conn.put_private(conn, :absinthe, Enum.into(opts, %{}))
   end
 
+  @doc """
+  Adds key-value pairs into Absinthe context.
+
+  ## Examples
+
+      iex> Absinthe.Plug.assign_context(conn, current_user: user)
+      %Plug.Conn{}
+  """
+  @spec assign_context(Plug.Conn.t(), Keyword.t() | map) :: Plug.Conn.t()
+  def assign_context(%Plug.Conn{private: %{absinthe: absinthe}} = conn, assigns) do
+    context =
+      absinthe
+      |> Map.get(:context, %{})
+      |> Map.merge(Enum.into(assigns, %{}))
```

Comment:
The merge order here is probably counter to what you want. `assigns` should be merged into the current context not the other way around, so that if you _do_ want to override an existing assign it works as expected.

```suggestion
    context =
      assigns
      |> Map.new
      |> Map.merge(Map.get(absinthe, :assigns, %{}))
```

Reply:
My understanding is that right wins.
```elixir
iex> Map.merge(%{foo: "bar"}, %{foo: "baz"})      
%{foo: "baz"}
```
So the current code un-pipelined
```elixir
context = Map.get(absinthe, :context, %{})
context = Map.merge(context, Map.new(assigns))
```
looks correct to me.

Reply:
You are correct! I got Map.merge and Enum.into backwards, my bad!